src/AboutDialog.cpp

```diff
@@ -223,6 +224,7 @@ void AboutDialog::CreateCreditsList()
    AddCredit(wxT("Mark Phillips"), developerFormat, roleContributor);
    AddCredit(wxT("Andr\u00E9 Pinto"), developerFormat, roleContributor);
    AddCredit(wxT("Jean Claude Risset"), composerFormat, roleContributor);
+   AddCredit(wxT("RuRo"), developerFormat, roleContributor);
```

Comment:
Are we okay with putting funny nicknames in the credits (instead of real names or at least pseudonyms)?  Does it look professional enough?  Should such handles be "quoted" Just asking an opinion @Tantacrul

Reply:
And do these individuals know they are being credited and have they agreed to being identified exactly so?

Reply:
If the answers are yes to all of that, ask me about someone who deserves an overdue credit too, though never identified to us by a real name.